cmd/apprepository-controller/controller.go

```diff
@@ -440,12 +441,12 @@ func newSyncJob(apprepo *apprepov1alpha1.AppRepository, kubeappsNamespace string
 			GenerateName:    cronJobName(apprepo) + "-",
 			OwnerReferences: ownerReferencesForAppRepo(apprepo, kubeappsNamespace),
 		},
-		Spec: syncJobSpec(apprepo),
+		Spec: syncJobSpec(apprepo, kubeappsNamespace),
 	}
 }
 
 // jobSpec returns a batchv1.JobSpec for running the chart-repo sync job
-func syncJobSpec(apprepo *apprepov1alpha1.AppRepository) batchv1.JobSpec {
+func syncJobSpec(apprepo *apprepov1alpha1.AppRepository, kubeappsNamespace string) batchv1.JobSpec {
```

Comment:
here (line 457) you also need to change the secretKeyRef?

Reply:
Yes, thanks.